WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs

```diff
@@ -361,6 +361,14 @@ public static void HandleGarrisonCovenantRenownSendCatchupState(Packet packet)
             packet.ResetBitReader();
             packet.ReadBit("CatchupState");
         }
+        
+        [Parser(Opcode.SMSG_COVENANT_PREVIEW_OPEN_NPC)]
+        public static void HandleGarrisonCovenantPreviewOpenNpc(Packet packet)
+        {
+            packet.ReadPackedGuid128("GUID");
+            packet.ReadUInt32("Index");
```

Comment:
```suggestion
            packet.ReadInt32("CovenantId");
```

Reply:
thank yes I couldn't find a name for it.

Reply:
Next time look in the handler. :)